doc/data-mining-library/source/reference/data.variable.rst

```diff
@@ -127,7 +127,17 @@ Time variables
 Time variables are continuous variables with value 0 on the Unix epoch,
 1 January 1970 00:00:00.0 UTC. Positive numbers are dates beyond this date,
 and negative dates before. Due to limitation of Python :py:mod:`datetime` module,
-only dates in 1 A.D. or later are supported.
+only dates in 1 A.D. or later are supported. Note that Orange's Table stores datetime 
+values as UNIX epoch (seconds from 1970-01-01), thus :obj:`Table.from_numpy` expects values in this format.
+Use `arr.astype("datetime64[s]")` to convert the array.
```

Comment:
I dropped this line since it is very case specific on users having DateTime in another unit (for example, minutes).